Source/WebKit/UIProcess/WebProcessActivityState.cpp

```diff
@@ -26,24 +26,34 @@
 #include "config.h"
 #include "WebProcessActivityState.h"
 
+#include "APIPageConfiguration.h"
 #include "RemotePageProxy.h"
 #include "WebPageProxy.h"
 #include "WebProcessProxy.h"
 
 namespace WebKit {
 
+#if PLATFORM(MAC)
+static Seconds webProcessSuspensionDelay(RefPtr<WebPageProxy> page)
```

Comment:
The parameter should not be a RefPtr, just use a raw pointer.

Reply:
Yeah, I had just worked through this after re-reading the safer C++ guidelines again.